api/extensions/ext_redis.py

```diff
@@ -1,23 +1,38 @@
 import redis
 from redis.connection import Connection, SSLConnection
+from rediscluster import RedisCluster
+import os
 
-redis_client = redis.Redis()
```

Comment:
Maybe we can set the `redis_client` to None, then this logic can be placed in the `init_app()` function.